lib/api/attributes.js

```diff
@@ -1,10 +1,12 @@
 var _ = require('lodash'),
+  static = require('../static'),
```

Comment:
Better yet, we could avoid defining the variable altogether:

```diff
-  static = require('../static'),
   // ...
-  text = static.text,
+  text = require('./static').text,
```

Besides side-stepping the naming problem, this makes the file a little more maintainable--it's clear that *only* the exported `text` method is used here (as opposed to "anything in the `static` module").